src/components/ble/MusicService.h

```diff
@@ -71,9 +71,9 @@ namespace Pinetime {
 
       uint16_t eventHandle {};
 
-      std::string artistName {"Waiting for"};
-      std::string albumName {};
-      std::string trackName {"track information.."};
+      std::string trackName {""};
+      std::string albumName {""};
```

Comment:
These initialisers are redundant, as the default constructor is the empty string
```suggestion
      std::string trackName;
      std::string albumName;
```